### PR TITLE
Feature/paragraph

### PR DESCRIPTION
**Pull request by yunusfirat**

### CODE REVIEW

**Review by daniele-zurico — COMMENTED** [review]
If you rebase your branch with release/0.8 you'll see that in the file 
`storybook/.preview` there's a section like this one:
```
'Typography', [
  'Heading',['Documentation', 'Live', 'Without style', 'Class based']
],
```
after 
```
 'Heading',['Documentation', 'Live', 'Without style', 'Class based']
```
add your component... I suppose will be something like:
```
 'Paragraph',['Documentation', 'Live', 'Without style', 'Class based']
```

File: src/paragraph/Paragraph.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+   * optional CSS class name
+   */
+  className?: string;
+  /**
+   * content of the paragraph
+   */
+  value: string | number;
+};
+
+export const Paragraph = ({ className, value, ...props }: ParagraphProps) => (
```
Comment (daniele-zurico) [inline src/paragraph/Paragraph.tsx:14]:
```suggestion
export const Paragraph = ({ className, value, props }: ParagraphProps) => (
```

File: stories/liveEdit/ParagraphLive.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const ParagraphDemo = `
+function ParagraphDemo() {
+
+  const style = {
+    color: 'red'
+  }
+
+  return (
+    <Paragraph
```
Comment (daniele-zurico) [inline stories/liveEdit/ParagraphLive.tsx:13]: list all the properties please something like:
<Paragraph className="" value="some value" props={{}}

File: stories/paragraph/Documentation.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+When you import the paragraph component without providing any className or style associated will looks as following:
+
+<Canvas of={ParagraphStories.Unstyled} />
+
+A more complex example with all the possible properties is:
+
+```js
+<Paragraph
+  label="Start Now"
+  value="This is the content of the paragraph."
```
Comment (daniele-zurico) [inline stories/paragraph/Documentation.mdx:19]: add props={{color:'red'}} or something similar

File: src/paragraph/__tests__/Paragraph.test.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    const { getByText } = render(<Paragraph className="paragraph" value="paragraph text" />);
+    const paragraph = getByText('paragraph text');
+    expect(paragraph).toBeTruthy();
+  });
+
+  it('Renders the given content in the paragraph.', () => {
+    render(<Paragraph  className="paragraph" value="paragraph text"/>);
+    expect(screen.getByText('paragraph text')).toBeInTheDocument();
+  });
+});
```
[the remark this answers is not in the document]
Comment (yunusfirat) [reply src/paragraph/__tests__/Paragraph.test.tsx:34]: I have added some more tests.

File: stories/liveEdit/ParagraphLive.tsx (hunk 2 above)
Comment (yunusfirat) [reply stories/liveEdit/ParagraphLive.tsx:9]: this has been resolved. The component has only 2 props: className and value. Any other arbitrary props can be passed in.

File: stories/paragraph/Documentation.mdx (hunk 3 above)
Comment (yunusfirat) [reply stories/paragraph/Documentation.mdx:19]: This has been resolved too.

File: src/paragraph/Paragraph.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+type ParagraphProps = React.HTMLAttributes<HTMLParagraphElement> & {
+  /**
+   * optional CSS class name
+   */
+  className?: string;
+  /**
+   * content of the paragraph
+   */
+  value: string | number;
+};
```
[the remark this answers is not in the document]
Comment (daniele-zurico) [reply src/paragraph/Paragraph.tsx:13]: yeah the problem is that the Props with the comment on top will appear in storybook as documentation ... if you don't put it the final user will not know that they can pass whatever they want... so the suggested solution to this problem is the following:
```tsx
  /**
   * Additional props/attributes
   */
  props?: React.HTMLAttributes<HTMLElement>;
};
export const Paragraph = ({ className, value, props }: ParagraphProps) => {
```
and everything will work exactly how you did

File: src/paragraph/__tests__/Paragraph.test.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Paragraph } from '../Paragraph';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+describe('Paragraph', () => {
+  it('should render the given content in the paragraph.', () => {
```
Comment (daniele-zurico) [inline src/paragraph/__tests__/Paragraph.test.tsx:7]: probably we just missing one test that check if the component is rendering.

**Review by daniele-zurico — APPROVED** [review]

**Pull request merged**